core/server/src/main/java/alluxio/web/UIWebServer.java

```diff
@@ -65,29 +70,43 @@ public UIWebServer(ServiceType service, InetSocketAddress address) {
     QueuedThreadPool threadPool = new QueuedThreadPool();
     int webThreadCount = Configuration.getInt(PropertyKey.WEB_THREADS);
 
-    mServer = new Server();
-    SelectChannelConnector connector = new SelectChannelConnector();
-    connector.setHost(address.getHostName());
-    connector.setPort(address.getPort());
-    connector.setAcceptors(webThreadCount);
-    mServer.setConnectors(new Connector[] {connector});
+    // Jetty needs at least (1 + selectors + acceptors) threads.
+    threadPool.setMinThreads(webThreadCount * 2 + 1);
+    threadPool.setMaxThreads(webThreadCount * 2 + 100);
```

Comment:
I believe Nate just ported the value we used to use for `maxThreads`, and I didn't see any reason to change it. If you think there is a better heuristic to use, I'm open to changing it.